packages/asc-ui/src/components/FilterTag/FilterTag.stories.tsx

```diff
@@ -29,14 +30,27 @@ storiesOf('Atoms/FilterTag', module)
         ref: React.useRef(null),
       },
     ]
+
+    const filterTagHandleClick = (nr: number): void => {
+      filterTagElements = filterTagElements
+        .filter(el => {
+          if (el.id !== nr) {
+            return el
+          }
+          return undefined
+        })
+        .map(el => el)
+    }
+
     return (
       <>
-        {myArray.map(element => (
+        {filterTagElements.map(element => (
           <FilterTag
-            ref={element.ref}
+            forwardRef={element.ref}
```

Comment:
if we put filterTagElements elements in a state, and we update the state with the new active / filtered out elements, we don't need a ref / forwardRef here